Tests/Tests/STPPaymentContext3DSTest.m

```diff
@@ -90,6 +99,25 @@ - (BOOL)threeDSParams:(STPSourceParams *)sourceParams matchCardSource:(STPSource
     return [expectedDict isEqualToDictionary:dict];
 }
 
+
+
+- (BOOL)precheckParams:(STPSourcePrecheckParams *)precheckParams matchPaymentContext:(STPPaymentContext *)paymentContext {
+
+    id<STPSourceProtocol> source = nil;
+    if ([paymentContext.selectedPaymentMethod conformsToProtocol:@protocol(STPSourceProtocol)]) {
+        source = (id<STPSourceProtocol>)paymentContext.selectedPaymentMethod;
+    }
+
+    STPSourcePrecheckParams *expectedParams = [STPSourcePrecheckParams new];
+    expectedParams.sourceID = source.stripeID;
+    expectedParams.paymentAmount = @(paymentContext.paymentAmount);
+    expectedParams.paymentCurrency = paymentContext.paymentCurrency;
```

Comment:
We should also check that `metadata` matches `paymentContext.sourceInformation.metadata`.
Also should verify additional parameters once we add a way to set those.